rotkehlchen/tests/external_apis/test_etherscan.py

```diff
@@ -82,19 +80,34 @@ def test_maximum_rate_limit_reached(temp_etherscan, **kwargs):  # pylint: disabl
 
     Regression test for https://github.com/rotki/rotki/issues/772"
     """
-    etherscan = temp_etherscan
-
-    etherscan_patch = patch_etherscan(etherscan)
+    etherscan_patch = patch_etherscan(
+        etherscan=temp_etherscan,
+        response_msg='Max rate limit reached, please use API Key for higher rate limit',
```

Comment:
is this the exact message etherscan currently sends?

Reply:
Kelsos shared it yesterday

```
{"status":"0","message":"NOTOK","result":"Max daily rate limit reached. 110000 (100%) of 100000 day/limit"}**
```